aidants_connect/settings.py

```diff
@@ -562,3 +562,13 @@ def getenv_bool(key: str, default: Optional[bool] = None) -> bool:
     "EMAIL_HABILITATION_ISSUER_EMAIL_ALREADY_EXISTS_SUBJECT",
     "Rappel de votre profil demandeur",
 )
+
+EMAIL_ORGANISATION_REQUEST_CRATION_FROM = os.getenv(
+    "EMAIL_ORGANISATION_REQUEST_CRATION_FROM", SUPPORT_EMAIL
```

Comment:
```suggestion
    "EMAIL_ORGANISATION_REQUEST_CREATION_FROM", SUPPORT_EMAIL
```

Bref, `s/CRATION/CREATION/` :)